test/unit/calculators/statutory_sick_pay_calculator_test.rb

```diff
@@ -1084,10 +1084,10 @@ class StatutorySickPayCalculatorTest < ActiveSupport::TestCase
               coronavirus_related: true,
             )
 
-            assert_equal 4, calc.days_paid
+            assert_equal 4, calc.days_paid # FIXME this should be false, we'll tackle it later
```

Comment:
"this should be 0", but I'll let it slide 😉

Reply:
Still applies because it'll be false when it fails ;)